README.md

```diff
@@ -68,25 +68,27 @@ Optimization simply use Ax.
 
 The latest release of BoTorch is easily installed either via
 [Anaconda](https://www.anaconda.com/distribution/#download-section) (recommended):
+
+First install PyTorch:
+```bash
+# Needed for OSX only, see below.
+conda install pytorch torchvision -c pytorch 
+```
+
+_Recommendation for MacOS users:_ PyTorch is a required dependency of BoTorch, and can be automatically installed via `pip`. However, we recommend installing PyTorch manually (as above) so that PyTorch is properly linked against MKL (a library that optimizes mathematical computation for Intel processors). This will result in up to an order-of-magnitude speed-up for Bayesian optimization, as at the moment, installing PyTorch from pip does not link against MKL.
+
+If you need CUDA on MacOS, or want to customize your installation (i.e. CPU-only option), please follow the [PyTorch installation instructions](https://pytorch.org/get-started/locally/) to build from source.
```

Comment:
So in more recent pytorch versions CUDA doesn't seem to be supported anymore on MacOS, but MPS (for Apple silicon) is: https://pytorch.org/get-started/locally/

We should update the readme here accordingly.

Reply:
Out of curiosity, has anyone tested BoTorch with MPS?

Reply:
I believe @sdaulton has. From browsing through its docs, MPS current still seems to be in its infancy in terms of supporting accelerated linear algebra, but it looks like they're working on closing that gap.